MapboxAndroidDemo/src/main/java/com/mapbox/mapboxandroiddemo/examples/snapshot/SnapshotShareActivity.java

```diff
@@ -61,12 +61,11 @@ public void onMapReady(@NonNull final MapboxMap mapboxMap) {
         mapboxMap.setStyle(Style.SATELLITE_STREETS, new Style.OnStyleLoaded() {
           @Override
           public void onStyleLoaded(@NonNull Style style) {
-            SnapshotShareActivity.this.style = style;
+            SnapshotShareActivity.this.mapboxMap = mapboxMap;
```

Comment:
shall we move this line to the `onMapReady()`?

Reply:
Sure, that's fine. Moved.